app/templates/components/ui-table/cell/events/view/videoroom/cell-stream-url.hbs

```diff
@@ -1,3 +1,19 @@
 {{#if @record}}
-  <a class="mt-2" href="{{href-to 'public.stream.view' (or @extraRecords.identifier @extraRecords.event.identifier) @record.slugName @record}}" target="_blank" rel="noopener">{{t 'Open'}}</a>
+  <div class="ui action input fluid">
+    <input type="text" name="private_link" value={{concat this.host '/e/' (or @extraRecords.identifier @extraRecords.event.identifier) '/video/' @record.slugName '/' @record.id}} placeholder={{concat this.host '/e/' (or @extraRecords.identifier @extraRecords.event.identifier) '/video/' @record.slugName '/' @record.id}} readonly>
+    <UiPopup @on="click" @content={{t "Link copied to clipboard"}} @position="bottom right">
+      <CopyButton @clipboardText={{concat this.host '/e/' (or @extraRecords.identifier @extraRecords.event.identifier) '/video/' @record.slugName '/' @record.id}} @class="ui black right labeled icon copy button">
```

Comment:
Repeating logic. Please create a computed property in component and put `concat this.host '/e/' (or @extraRecords.identifier @extraRecords.event.identifier) '/video/' @record.slugName '/' @record.id}} placeholder={{concat this.host '/e/' (or @extraRecords.identifier @extraRecords.event.identifier) '/video/' @record.slugName '/' @record.id` there